src/Common/CommonServices.ts

```diff
@@ -32,7 +32,12 @@ export const enum CommonServiceIds {
     /**
      * Exposes project-related information from the current page
      */
-    ProjectPageService = "ms.vss-tfs-web.tfs-page-data-service"
+    ProjectPageService = "ms.vss-tfs-web.tfs-page-data-service",
+
+    /**
+     * Provides a way to search for identities.
+     */
+    IIVssIdentityService = "ms.vss-features.IIVssIdentityService"
```

Comment:
@thomabr The registered service name here is consistent with all our other remote ones -- could this be ms.vss-features.identity-service?

Reply:
I was going to talk to you about this. This is the name currently being registered in our product for the service. What sort of back-compat needs to be done before changing this?